Aliucord/src/main/java/com/aliucord/coreplugins/Polls.kt

```diff
@@ -346,19 +346,20 @@ internal class Polls : CorePlugin(Manifest("Polls")) {
             val ctx = flexInputFragment.requireContext()
             val pages = flexInputFragment.r.toMutableList()
 
-            val channel = StoreStream.getChannelsSelected().selectedChannel
-                ?: return@after
-            val isPrivate = ChannelUtils.B(channel)
-            val isSystem = ChannelUtils.E(channel)
-            val permissions = StoreStream.getPermissions().permissionsByChannel[channel.id]
-            val hasPermission = PermissionUtils.can(SEND_POLLS_PERMISSION, permissions)
-            val permitted = (isPrivate && !isSystem) || hasPermission
-
-            if (!permitted) return@after
+            if (ManagerBuild.hasPatches("1.3.1")) {
```

Comment:
Can you add a small comment here with a link to this pr or a small explanation of what's going on